pkg/btf/btf.go

```diff
@@ -123,6 +123,11 @@ func FindBtfFuncParamFromHook(hook string, argIndex int) (*btf.FuncParam, error)
 	}
 
 	if err = spec.TypeByName(hook, &hookFn); err != nil {
+		if strings.HasPrefix(hook, "bpf_lsm_") {
```

Comment:
I think it is acceptable to such a check instead of passing a boolean directly to the function. I wanted to keep the function resilient